emoji-copy@felipeftn/extension.js

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright 2017-2022 Romain F. T.
```

Comment:
You probably shouldn't remove the copyright notice of the original author, but just add your own copyright notice

Reply:
Nice catch! Thank you!! :tada:

Reply:
You're welcome :)